diff --git a/node_modules/appdmg/lib/appdmg.js b/node_modules/appdmg/lib/appdmg.js
--- a/node_modules/appdmg/lib/appdmg.js
+++ b/node_modules/appdmg/lib/appdmg.js
@@ -6,7 +6,7 @@
 
 const async = require('async')
 const DSStore = require('ds-store')
-const sizeOf = require('image-size')
+const { imageSizeFromFile } = require('image-size/fromFile')
 const validator = require('is-my-json-valid')
 const parseColor = require('parse-color')
 
@@ -286,12 +286,10 @@
   pipeline.addStep('Reading background dimensions', function (next) {
     if (!global.opts.background) return next.skip()
 
-    sizeOf(resolvePath(global.opts.background), function (err, value) {
-      if (err) return next(err)
-
+    imageSizeFromFile(resolvePath(global.opts.background)).then(function (value) {
       global.bkgsize = [value.width, value.height]
       next(null)
-    })
+    }, next)
   })
 
   /**
